main.go

```diff
@@ -119,3 +130,43 @@ func printVersion() {
 	setupLog.Info(fmt.Sprintf("Git Commit: %s", version.GitCommit))
 	setupLog.Info(fmt.Sprintf("Build Date: %s", version.BuildDate))
 }
+
+// Creates the patterns operator configmap
+// This will include configuration parameters that
+// will allow operator configuration
+func createGitOpsConfigMap() error {
+	// LRC - Declared as global const
+	// operatorConfigFile := "patterns-operator-config"
+	// operatorNamespace := "openshift-operators"
+
+	config, _ := ctrl.GetConfig()
+	clientset, _ := kubernetes.NewForConfig(config)
+	configMapData := map[string]string{
```

Comment:
Same here, if you have a separate type + the default config map, you can totally reuse it here, so everything is defined in a single file

Reply:
Done.